include/SDL3/SDL_gpu.h

```diff
@@ -621,9 +624,10 @@ typedef struct SDL_GPUSamplerCreateInfo
     SDL_GPUSamplerAddressMode addressModeV;
     SDL_GPUSamplerAddressMode addressModeW;
     float mipLodBias;
-    SDL_bool anisotropyEnable;
     float maxAnisotropy;
+    SDL_bool anisotropyEnable;
     SDL_bool compareEnable;
+    Uint16 padding;
```

Comment:
This should be two Uint8 padding fields. This _should_ pack the same, but I find it's better to be explicit.

Reply:
Also, if you require that structs be zero initialized, these can become additional bool fields in the future without changing the ABI.